syntax = "proto3";

message AudioChunk {
    bytes pcm = 1;         
}

message TranscriptionResponse {
    string transcript = 1; 
}
